scripts/mrbind/install_mrbind_ubuntu.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Downloads the source code for MRBind and builds it at `~/mrbind/build`.
+
+set -euxo pipefail
+
+sudo apt install -y git
+
+MRBIND_DIR=~/mrbind
```

Comment:
You can make it overridable with
```bash
MRBIND_DIR=${MRBIND_DIR:-$HOME/mrbind}
```
However I don't know which other scripts can be affected by it.

Reply:
Ok. Will also do it in `install_mrbind_macos.sh`.